webui/react/src/shared/bin/test.py

```diff
@@ -7,7 +7,7 @@
 parser = argparse.ArgumentParser()
 
 SHARED_WEB_REPO='https://github.com/determined-ai/shared-web'
-SAAS_REPO='https://github.com/determined-ai/saas'
+SAAS_REPO = "ssh://git@github.com/determined-ai/saas"
```

Comment:
don't know if this change was actually needed. will check.

Reply:
probably needed as it's a private repo. could we keep the spacing and quotations the same?

Reply:
leave `ssh` but change quotation marks?

or change to `https` and quotation marks and see if it works?

Reply:
if as is, it also works locally for people as well then either one sounds good.